apps/machine/microblaze_generic/platform_info.c

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright (c) 2020 Xilinx, Inc.
+ * All rights reserved.
```

Comment:
squash on one line to have one line per copyright. Please check also the other files of the PR.

Reply:
will fix